### ISSUE TITLE
[MINOR BUG] Newly Created or Filled Elements Include Modal Language

### ISSUE THREAD

**Issue by sigrunixia** [carlonicora/obsidian-rpg-manager#236]
## Scope

Element Creation (Probably Hierarchy Creation as well but I did not do that this evening).

### To Reproduce

Newly created or filled elements are created with the following format in their templates:

`Element-Type+Model+Notes`

```
### Non Player CharacterModel Notes
- 

### Non Player CharacterModel Story
```
### Expected behaviour

`Element-Type+Notes`
```
### Non Player Character Notes
- 

### Non Player Character Story
```

### Additional context
Minor Bug
FC-Calendar functionality is not enabled

**Comment by carlonicora** [comment]
@sigrunixia i am not sure I follow you.
I don’t understand the difference from current to expected.

**Comment by sigrunixia** [comment]
I will get better at not posting these RIGHT before I go to sleep, or at least including screenshots. :)

#### Pre-Patch Generation of Elements
![pre-patch](https://user-images.githubusercontent.com/68425372/201403579-9a571632-6893-48ed-af1b-784bf31fdbb1.png)

#### Post-Patch Generation of Elements
![post-patch](https://user-images.githubusercontent.com/68425372/201403639-a5d4117f-d558-43a6-8127-2b26ad79dc4d.png)

**Comment by carlonicora** [comment]
Released in 3.2.1
